src/test/java/org/opensearch/geospatial/ip2geo/action/GetDatasourceRequestTests.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.geospatial.ip2geo.action;
+
+import org.opensearch.common.io.stream.BytesStreamInput;
+import org.opensearch.common.io.stream.BytesStreamOutput;
+import org.opensearch.geospatial.GeospatialTestHelper;
+import org.opensearch.geospatial.ip2geo.Ip2GeoTestCase;
+
+public class GetDatasourceRequestTests extends Ip2GeoTestCase {
+    public void testStreamInOutWithEmptyNames() throws Exception {
```

Comment:
We can @SneakyThrow instead of adding throws Exception in method signature. Little cleaner as we don't care what exceptions come from tests.

Reply:
Will work on it later for the entire repo. https://github.com/opensearch-project/geospatial/issues/287